src/fci/fci_vector_rdm.cc

```diff
@@ -933,4 +933,63 @@ void FCIVector::rdm_test() {
     delete[] Ia;
     delete[] Ib;
 }
+
+/**
+ * Compute the ab two-particle density matrix for a given wave function
+ * @param alfa flag for alfa or beta component, true = aa, false = bb
```

Comment:
Should this be "alpha"? I see you use "alfa" elsewhere, so this has to be deliberate.

Reply:
Probably because "alpha" is 1 letter longer than "beta".

Reply:
Ah, yes, "alfa" is intentional here. Alfa is the Italian spelling and gives lovely formatted code since it matches the four characters of "beta". One day we will need to run a search and replace "alfa -> alpha" on all the code.